### ISSUE TITLE
Responsive design

### ISSUE THREAD

**Issue by kaleyh** [legomushroom/iconmelon#1]
Please find a screenshot from my Nexus 4.
![2013-11-28 05 15 56](https://f.cloud.github.com/assets/3038853/1637471/46317f30-57ec-11e3-8608-a91e14431844.png)

**Comment by legomushroom** [comment]
Hey! What wrong is here? "No icons" issue or you want to see a mobile version? Thanks!

**Comment by kaleyh** [comment]
I was hoping to see a mobile (responsive) version.

**Comment by legomushroom** [comment]
Well, it probably needs a mobile version but I can't imagine that somebody will load an icon set from a mobile device. What is your thoughts about it?

**Comment by kaleyh** [comment]
I agree that an icon set would likely never be loaded from a mobile device;
although it would be useful to have a mobile design that allowed for an
easy way to view what was available.

**Comment by sbrandwoo** [comment]
I am currently here on a mobile device. While you won't do much icon work on a mobile, you do still follow twitter links and such, so marketing the icons is still important. 

It does work perfectly fine for my on Chrome Android though, gives me the desktop site but all the icons show up.
